webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/RestTestSetUp.scala

```diff
@@ -934,21 +947,22 @@ class RestTestSetUp {
       mockUserManagement.tenantsService,
       () => mockUserManagement.providerRoleExtension,
       () => mockUserManagement.authBackendProviders
-    )
+    ),
+    infoApi
   )
 
   val apiVersions: List[ApiVersion] = {
-    ApiVersion(14, deprecated = true) ::
-    ApiVersion(15, deprecated = true) ::
-    ApiVersion(16, deprecated = true) ::
     ApiVersion(17, deprecated = true) ::
-    ApiVersion(18, deprecated = false) ::
-    ApiVersion(19, deprecated = false) ::
+    ApiVersion(18, deprecated = true) ::
+    ApiVersion(19, deprecated = true) ::
     ApiVersion(20, deprecated = false) ::
+    ApiVersion(21, deprecated = false) ::
     Nil
```

Comment:
I cleaned up test version, forgot to do it in the previous PR adding new version.